tests/func/ldap2pg.acl.yml

```diff
@@ -41,7 +41,7 @@ sync_map:
 - grant:
     # Grant ro to some developers
     database: appdb
-    schema: appns
+    schema: [appns]
```

Comment:
Should we change keys to plural? Or perhaps accept both spellings?

Reply:
@cbandy yes, both are accepted. Like `names` in `role`/`roles`.